mappings/net/minecraft/entity/decoration/AbstractDecorationEntity.mapping

```diff
@@ -6,8 +6,9 @@ CLASS aph net/minecraft/entity/decoration/AbstractDecorationEntity
 		ARG 1 type
 	METHOD <init> (Lahq;Lbdf;Les;)V
 		ARG 1 world
-	METHOD a copyEntityData (Lahm;)V
+	METHOD a getBreakEvents (Lahm;)V
```

Comment:
get... and it returns void?

Reply:
Maybe fireBreakEvents/firePlaceEvents could be better?

Reply:
Better, but it still sounds very off. Why not just onBreak/onPlace?